### ISSUE TITLE
Not compatible with Aeson 1.0

### ISSUE THREAD

**Issue by LeviSchuck** [klappvisor/haskell-telegram-api#64]
There are no upper bound limits on many dependencies on this package.

`aeson` is one of them, and it fails to compile when the system downloads the latest package.

```
Preprocessing library telegram-api-0.4.3.1...
[1 of 7] Compiling Web.Telegram.API.Bot.JsonExt ( src/Web/Telegram/API/Bot/JsonExt.hs, dist/dist-sandbox-fea0daa/build/Web/Telegram/API/Bot/JsonExt.o )

src/Web/Telegram/API/Bot/JsonExt.hs:16:58:
    Expecting one more argument to ‘Rep a’
    The first argument of ‘GToJSON’ should have kind ‘*’,
      but ‘Rep a’ has kind ‘* -> *’
    In the type signature for ‘toJsonDrop’:
      toJsonDrop :: forall a. (Generic a, GToJSON (Rep a)) =>
                    Int -> a -> Value

src/Web/Telegram/API/Bot/JsonExt.hs:23:50:
    Expecting one more argument to ‘Rep a’
    The first argument of ‘GFromJSON’ should have kind ‘*’,
      but ‘Rep a’ has kind ‘* -> *’
    In the type signature for ‘parseJsonDrop’:
      parseJsonDrop :: forall a. (Generic a, GFromJSON (Rep a)) =>
                       Int -> Value -> Parser a
cabal: Error: some packages failed to install:
```

Either ensure compatibility, or put an upper limit to avoid this and migrate later.

**Comment by klappvisor** [comment]
@LeviSchuck Hi, thank you for opening that issue. Will be fixed soon

**Comment by timjb** [comment]
This has been fixed by 0b0e29137b514b15ffe7fc4d0da32d9b15f160e5, right?

**Comment by klappvisor** [comment]
@timjb Yes, I think so
